src/components/Rating/Rating.js

```diff
@@ -161,7 +170,9 @@ Rating.propTypes = {
   /** Don't allow input if set to true. */
   readOnly: PropTypes.bool,
   /** The value of the rating bar. */
-  value: PropTypes.number
+  value: PropTypes.number,
+  /** The component used for the root node. Either a string to use a DOM element or a component.*/
```

Comment:
I might be nit-picking, but the propTypes used to be sorted alphabetically here (as well as in the README).

Reply:
it's all good mate, every little bit of readability counts